app/controllers/course/announcements_controller.rb

```diff
@@ -4,6 +4,8 @@ class Course::AnnouncementsController < Course::ModuleController
   def index #:nodoc:
     @announcements = @announcements.includes(:creator).sorted_by_sticky.sorted_by_date
     @announcements = @announcements.page(params[:page])
+    unread = @announcements.unread_by(@current_user)
```

Comment:
I believe `current_user` is a function. Use the function instead of the member access where possible.